todo.py

```diff
@@ -55,6 +56,15 @@ def delete_task():
         print(f"Task {task_num} deleted: {removed_task['task']}")
     except (ValueError, IndexError):
         print("Invalid task number.")
+
+def put_task():
+    task_index = int(input("Enter the number of the task to change: "))
```

Comment:
!suggest change the task_index variable name to task_number in all the occurences